.cloudbuild/std-build.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - name: maven:3.6-jdk-11
```

Comment:
It's probably not a good idea to mix our maven wrapper `./mvnw` with a separate install of maven.

Reply:
You can use the base image instead (`openjdk:11-jdk`) here and below and just use the wrapper here.

Reply:
Done.